src/sdk/examples/AutoCreateAccountTransferTransactionExample.cc

```diff
@@ -1,8 +1,8 @@
 /*-
  *
- * Hedera C++ SDK
+ * Hiero C++ SDK
  *
- * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2020 - 2024 Hiero
```

Comment:
I'm not sure about the renaming in the licenses. Can someone give some further details?